build_tools/benchmarks/comparisons/setup_desktop.sh

```diff
@@ -0,0 +1,110 @@
+#!/bin/bash
+
+# Install Bazel. From https://www.tensorflow.org/install/source
+npm install -g @bazel/bazelisk
+
+# Create root dir.
+ROOT_DIR=/tmp/mobilebert_benchmarks
+mkdir ${ROOT_DIR}
+mkdir ${ROOT_DIR}/models
+mkdir ${ROOT_DIR}/models/tflite
+mkdir ${ROOT_DIR}/models/iree
+mkdir ${ROOT_DIR}/test_data
+mkdir ${ROOT_DIR}/output
+
+wget https://storage.googleapis.com/iree-model-artifacts/tflite_squad_test_data.zip -O /tmp/tflite_squad_test_data.zip
+unzip /tmp/tflite_squad_test_data.zip -d ${ROOT_DIR}/test_data/
+wget https://storage.googleapis.com/iree-model-artifacts/mobilebert_float_384_gpu.tflite -O ${ROOT_DIR}/models/tflite/mobilebert_float_384_gpu.tflite
+
+# Build IREE source.
+SOURCE_DIR=/tmp/github
+mkdir ${SOURCE_DIR}
+cd ${SOURCE_DIR}
+
+#git clone https://github.com/google/iree.git
+git clone https://github.com/mariecwhite/iree.git
+
+cd iree
+git checkout origin/comparisons
+git submodule update --init
+# Below is only needed if CUDA is not setup.
+#export IREE_CUDA_DEPS_DIR="/usr/local/iree_cuda_deps"
```

Comment:
That seems like setup that someone should have in their environment external to this script. I would just delete this bit

Reply:
Removed.